common/utils/analytics.js

```diff
@@ -53,6 +53,10 @@ export default ({ category, contentType, pageState, featuresCohort }: Props) =>
   if (category) {
     ReactGA.set({'dimension2': category});
   };
+
+  if (pageState.serial) {
+    ReactGA.set({'dimension3': pageState.serial});
```

Comment:
how we were reporting this in the past was `${series.id}${series.title}` to make it easier for @hthair to read.

Reply:
Wait, got it, we do this down here.

Feels a bit disjointed the GA stuff being done in the article, then past down the pipes - given that we have series on Events too, maybe page state just takes a `series: ?ArticleSeries | EventSeries`?

Reply:
It was `${series.title}:${series.id}` – that's what this property will be if it's there.